manual/README.md

```diff
@@ -1,17 +1,295 @@
 ## Manual
 
-This part of the documentation is organized into sub-sections covering
-specific topics.
+### Quick start
+
+Here's a short program that connects to Cassandra and executes a query:
+
+```java
+Cluster cluster = null;
+try {
+    cluster = Cluster.builder()                                                    // (1)
+            .addContactPoint("127.0.0.1")
+            .build();
+    Session session = cluster.connect();                                           // (2)
+
+    ResultSet rs = session.execute("select release_version from system.local");    // (3)
+    Row row = rs.one();
+    System.out.println(row.getString("release_version"));                          // (4)
+} finally {
+    if (cluster != null) cluster.close();                                          // (5)
+}
+```
+
+1. the [Cluster] object is the main entry point of the driver. It holds the known state of the actual Cassandra cluster
```

Comment:
Ah, you sort of cover that below under Cluster initialization, so maybe that isn't needed here.